index.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] It is nice to tell why students should change code but instead of giving like an order, it will be better writing as suggestions

> It will be better if you replace all `<header>` tags with any of the `<h1...h6>` tags. This will improve our use of semantics elements for easy accessibility.